modules/configuring-a-proxy-after-installation-cli.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-wide-proxy.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-a-proxy-after-installation-cli_{context}"]
+= Configuring a proxy after installation using the CLI
+
+You can use the {product-title} (ROSA) CLI (`rosa`) to add a cluster-wide proxy configuration to an existing ROSA cluster in a Virtual Private Cloud (VPC).
+
+You can also use `rosa` to update an existing cluster-wide proxy configuration. For example, you might need to update the network address for the proxy or replace the additional trust bundle if any of the certificate authorities for the proxy expire.
+
+[IMPORTANT]
+====
+The cluster applies the proxy configuration to the control plane and compute nodes. While applying the configuration, each cluster node is temporarily placed in an unschedulable state and drained of its workloads. Each node is restarted as part of the process.
+====
+
+.Prerequisites
+
+* You have installed and configured the latest ROSA (`rosa`) and OpenShift (`oc`) CLIs on your installation host.
```

Comment:
is installing `oc` still a prerequisite for ROSA?

Reply:
This is a prerequisite for the verification step in this procedure, unless we update that step to use the `rosa` CLI instead. I will update this depending on how that develops.